chia/rpc/wallet_rpc_api.py

```diff
@@ -124,6 +124,7 @@ def get_routes(self) -> Dict[str, Callable]:
             # NFT Wallet
             "/nft_mint_nft": self.nft_mint_nft,
             "/nft_get_current_nfts": self.nft_get_current_nfts,
```

Comment:
We don't appear to be using `nft_get_current_nfts` anywhere, and your new API appears to supercede it. Can you delete `nft_get_current_nfts`?